### ISSUE TITLE
Datepicker doesn't support aria-describedby attribute

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#494]
### Describe the bug

```html
<p-datepicker aria-describedby="datepicker-information" />
<small id="datepicker-information">Additional information</small>
```
`aria-describedby` doesn't get applied to the child <input>.
`attr.aria-describedby` also doesn't fix the issue.

### Pull Request Link

https://github.com/primefaces/primeng/pull/1155

### Reproducer

https://stackblitz.com/edit/nc7m654p?file=src%2Fapp%2Fdatepicker-basic-demo.html

### Environment

Newly created project

### Angular version

19.0.1

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Create new Angular project with PrimeNG
2. Add the attribute `aria-describedby` or `[attr.aria-describedby]` to a `p-datepicker` component.
3. Look at the `input` element of the datepicker and see that there is no `aria-decribedby` association.

### Expected behavior

The datepicker `input` should be associated with `aria-describedby`, so that a screen reader can read the description when `p-datepicker` is focused.

> Migrated from `primefaces/primeng#18264` — originally by `@Will-Smith-007` on 2025-05-08

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-11-05T00:20:20.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

**Comment by geromegrignon** [comment]
*[Originally posted by `@Will-Smith-007` on 2026-05-31T18:46:38.000Z]*

This issue is still relevant and will be fixed by #18265. The PR was updated.
